decidim-core/db/migrate/20180221101934_fix_nickname_index.rb

```diff
@@ -6,14 +6,13 @@ class User < ApplicationRecord
   end
 
   def change
-    Decidim::User.where(nickname: nil)
-                 .where(deleted_at: nil)
-                 .where(managed: false)
-                 .find_each { |u| u.update(nickname: Decidim::User.nicknamize(u.name)) }
+    User.where(nickname: nil)
```

Comment:
How does this solve the issue? Is it something about namespaces and class reloading? 😕 

Won't this clash if the app, for some reason, has a `User` model? I'm thinking maybe there's a way to force Rails to load all classes before running migrations...

Reply:
This is a technique we've been using for a while. You try to use plain SQL but if you need AR, you use a dummy model scoped to the migration. In this case, I'm loading the class `FixNicknameIndex::User`. The idea is that if I load the real class now, that version will stay loaded, and that version is incomplete because we're in the middle of migrations... So we don't use the real classes during migrations.

Reply:
Oh, I didn't see the class is defined on top 😅 (GitHub didn't render that part)

👍